lib/Dialect/OM/Transforms/FreezePaths.cpp

```diff
@@ -118,6 +120,8 @@ LogicalResult PathVisitor::processPath(Location loc, hw::HierPathOp hierPathOp,
     auto *op = target.getOp();
     // Get the verilog name of the target.
     auto verilogName = op->getAttrOfType<StringAttr>("hw.verilogName");
+    if (!verilogName && getOpName)
+      verilogName = getOpName(op);
```

Comment:
My only nitpick is this is only invoked as a fallback when hw.verilogName is not set, and calling it getOpName seems to me like this is the main name-getting function, not a fallback. Do with this feedback what you will... I think this is fine, but I might call it something like `getOpNameFallback` or something like that.